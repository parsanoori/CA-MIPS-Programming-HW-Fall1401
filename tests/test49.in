82
908202711
329856888
601826025
341325391
215652739
668319525
47219236
212497340
243870789
1031800149
17259794
386233911
1061680745
992597265
1038636162
68563429
252104408
468121226
485806394
263452825
818835291
267740722
281111197
545295125
523351332
889364410
356375075
704943822
1062514273
199920083
746042873
329652396
244007730
667078925
853431361
110594977
133766186
246691014
23152177
895518944
524686132
789887906
714257579
47428585
772789529
663974919
781181033
913319731
161283160
330028080
125573577
539619189
898972461
347377409
224316727
91859
929109402
1038221891
219560172
680621061
790758705
874767454
413747688
866230786
483810408
1031382266
566478866
152226879
689379109
614694989
639705079
749515232
198104786
905076155
598418897
359406933
301546724
224168057
311408094
503646718
695815806
542553167