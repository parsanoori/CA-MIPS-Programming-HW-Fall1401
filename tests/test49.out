91859
17259794
23152177
47219236
47428585
68563429
110594977
125573577
133766186
152226879
161283160
198104786
199920083
212497340
215652739
219560172
224168057
224316727
243870789
244007730
246691014
252104408
263452825
267740722
281111197
301546724
311408094
329652396
329856888
330028080
341325391
347377409
356375075
359406933
386233911
413747688
468121226
483810408
485806394
503646718
523351332
524686132
539619189
542553167
545295125
566478866
598418897
601826025
614694989
639705079
663974919
667078925
668319525
680621061
689379109
695815806
704943822
714257579
746042873
749515232
772789529
781181033
789887906
790758705
818835291
853431361
866230786
874767454
889364410
895518944
898972461
905076155
908202711
913319731
929109402
992597265
1031382266
1031800149
1038221891
1038636162
1061680745
1062514273